crates/mun_compiler/src/driver.rs

```diff
@@ -107,14 +106,22 @@ impl Driver {
 
     /// Emits all diagnostic messages currently in the database; returns true if errors were
     /// emitted.
-    pub fn emit_diagnostics(&self, writer: &mut impl WriteColor) -> Result<bool, failure::Error> {
+    pub fn emit_diagnostics(&self) -> Result<bool, failure::Error> {
         let mut has_errors = false;
+        let dlf = DisplayListFormatter::new(true, false);
         for file_id in self.db.source_root(WORKSPACE).files() {
             let diags = diagnostics(&self.db, file_id);
-            for diagnostic in diags.iter() {
-                diagnostic.emit(writer, &self.db, file_id)?;
-                if diagnostic.level == Level::Error {
-                    has_errors = true;
+            for diagnostic in diags {
+                let dl = DisplayList::from(diagnostic.clone());
+                eprintln!("{}", dlf.format(&dl));
+                if let Some(annotation) = diagnostic.title {
+                    #[allow(clippy::single_match)]
```

Comment:
I'd rather that you not ignore this clippy warning, but do as it says by using `if let`

Reply:
> I'd rather that you not ignore this clippy warning, but do as it says by using `if let`

I ignore this clippy warning because in the future here also can be matching with `AnnotationType::Warning`, `AnnotationType::Info` and etc. Do you anyway recommend for now change that matching to `if let`?